<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>591</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QComboBox" name="StudAssigComboBox">
         <item>
          <property name="text">
           <string>Choose data</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Students</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Assignments</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="dataList"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeButton">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="updateButton">
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLineEdit" name="idText">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameDescrText"/>
         </item>
         <item>
          <widget class="QLineEdit" name="groupDateText"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="displayLabel">
         <property name="text">
          <string>Situation:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="displayList">
         <item>
          <property name="text">
           <string>Students and their Assignments</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Students ordered by average grade of an Assignment</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Students late in handling Assignments</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>best Students</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="displayButton">
         <property name="text">
          <string>Display</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="assignmentButton">
         <property name="text">
          <string>Give Assignment</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="gradeButton">
         <property name="text">
          <string>Give Grade</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="undoButton">
         <property name="text">
          <string>Undo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="redoButton">
         <property name="text">
          <string>Redo</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>625</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
